packages/commitlint-config-jira/src/ingores/releases.ts

```diff
@@ -0,0 +1,5 @@
+import { Matcher } from '@commitlint/types';
+
+const RELEASE_REGEX = new RegExp('Releas(e|ing) v?\\d+\\.\\d+\\.\\d+');
+
+export const releases: Matcher = (commit: string) => RELEASE_REGEX.test(commit);
```

Comment:
🍹 Do we want to add some tests for this, to make sure it does what it should do?

Reply:
Let me throw up another PR for this, since it'll include adding jest to package.